### ISSUE TITLE
Ffmpeg 5 is broken on debian buster

### ISSUE THREAD

**Issue by mirolm** [jellyfin/jellyfin-ffmpeg#135]
**Describe The Bug**

After the update of latest jellyfin-ffmpeg my jellyfin stopped to work. Upon inspection of logs it looked like this:

/usr/lib/jellyfin-ffmpeg/ffmpeg: error while loading shared libraries: libglslang.so.11: cannot open shared object file: No such file or directory

If you go ldd /usr/lib/jellyfin-ffmpeg/ffmpeg it shows:

	libglslang.so.11 => not found
	libHLSL.so => not found

**Steps To Reproduce**

1. Update to latest jellyfin-ffmpeg
2. Play file any file
3. Check jellyfin logs

**Expected Behavior**

Jellyfin should play videos without issue. I reverted back to jellyfin-ffmpeg_4.4.1-4-buster_amd64.deb and it fixed the issue.

**System (please complete the following information):**

- **Platform:** Linux
- **Operating System:** Debian Buster 10
- **Virtualization**: No
- **Jellyfin Version:** 10.7.7
- **Jellyfin-FFmpeg Version:** jellyfin-ffmpeg_5.0.1-1-buster_amd64
- **Hardware Acceleration:** Nvidia
- **CPU & GPU Model**: E3-1230 v5, Nvidia T600

**MediaInfo**

**FFmpeg Logs**

**Additional Context**

**Comment by nyanmisaka** [comment]
Upgrading to the hot fix version 5.0.1-2 should fix this issue. Other distros are not affected.

**Comment by nodiscc** [comment]
@nyanmisaka 

> Upgrading to the hot fix version 5.0.1-2 should fix this issue

There is no such version in the APT repository

```
$ sudo apt update
$ apt policy jellyfin-ffmpeg
jellyfin-ffmpeg:
  Installed: 4.4.1-4-bullseye
  Candidate: 4.4.1-4-buster
  Version table:
     4.4.1-4-buster 500
        500 https://repo.jellyfin.org/debian buster/main amd64 Packages
 *** 4.4.1-4-bullseye 500
        500 https://repo.jellyfin.org/debian bullseye/main amd64 Packages
        100 /var/lib/dpkg/status
```

(I downgraded to 4.4 when I noticed the problem with 5.0, since then I can't see `jellyfin-ffmpeg_5.0.1-1-buster_amd64` anymore either - i guess it was removed from the repo?)

**Comment by crobibero** [comment]
Try `jellyfin-ffmpeg5`
